src/app/_components/review/review-modal/ReviewRegisterModal.tsx

```diff
@@ -85,14 +85,22 @@ function ReviewRegisterModal({ gym_name, mutate }: ReviewRegisterProps) {
     closeMode();
   };
 
+  let header = null;
+  let submit = null;
+  if (state === 'create') {
+    header = '리뷰 작성';
+    submit = '작성 완료';
+  } else if (state === 'fix') {
+    header = '리뷰 수정';
+    submit = '수정 완료';
+  }
```

Comment:
```suggestion
  if (state === 'create') {
    header = '리뷰 작성';
    submit = '작성 완료';
  } else {
    header = '리뷰 수정';
    submit = '수정 완료';
  }
```

현재 컴포넌트에서 리뷰 작성과 수정 케이스 외에는 존재하지 않을 것 같아서 `if-else` 문이 적합해 보입니다 :>

Reply:
state 값이 공백이 될 수도 있고 여러 형태가 있어서 다음과 같이 해두었습니다. 실제로 모드 종료를 할경우 공잭으로 표시됩니다!